v2/app/testdata/pubsub_err_ordering_attribute_missing.txt

```diff
@@ -0,0 +1,42 @@
+# Verify that pub sub is parsed
+! parse
+err 'PubSub message attributes must not be prefixed with "encore".'
```

Comment:
This seems wrong; there is no such attribute

Reply:
The v2 parser tests don't actually look at the `err` flag, so I've removed this (it was copy/pasted from a previous test)